src/transformers/cache_utils.py

```diff
@@ -82,6 +88,129 @@ def seen_tokens(self):
             return None
 
 
+@dataclass
+class CacheConfig:
+    """
+    Configuration class for quantized cache settings.
```

Comment:
If this is a config, it should be `QuantizedCacheConfig` that inherits form a more general CacheConfig

Reply:
Yes, makes sense as we want to start using CacheConfig at some point to hold all cache related args, including the cache_implementation